src/deploy/osp_deployer/deployer.py

```diff
@@ -215,7 +221,8 @@ def deploy():
         director_vm.setup_net_envt()
         director_vm.configure_dhcp_server()
         director_vm.node_discovery()
-        director_vm.configure_idracs()
+        if args.skip_idrac_config is False:
+            director_vm.configure_idracs()
```

Comment:
I think the way this is implemented will be a bit confusing for users, since if skip_idrac_config is True, then it will skip NIC configuration too regardless of what skip_nic_config is set to.  I think it would be better to only skip config_idracs.py if skip_idrac_config is True so that the 2 flags work independently.

Reply:
Makes sense, will do.